types/body-scroll-lock/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for body-scroll-lock 2.6
+// Project: https://github.com/willmcpo/body-scroll-lock (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: kimcoder <https://github.com/kimcoder>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface BodyScrollOptions {
+    reserveScrollBarGap?: boolean;
+    allowTouchMove?: () => void;
```

Comment:
From looking at https://github.com/willmcpo/body-scroll-lock#simple and https://github.com/willmcpo/body-scroll-lock#more-complex
This seems like should be `allowTouchMove?: (el: HTMLElement) => void` instead.